roles/variables/defaults/main.yml

```
@@ -1327,6 +1343,103 @@ control_center_custom_properties: {}
 ### Boolean used for disabling of systemd service restarts when rootless install is executed
 control_center_skip_restarts: "{{ skip_restarts }}"
 
+# Control Center Next Gen Variables
+
+### Default Control Center config prefix. Only valid to customize when installation_method: archive
+control_center_next_gen_config_prefix: "{{ config_prefix }}/confluent-control-center"
```

what are doing in case someone tries to setup c3 and c3pp colocated ? 
Are we failing early or it can mess up both later?

---

co-location is not supported because of limitation on new package having same internal names limitation